src/styles/GlobalStyle.css

```
@@ -0,0 +1,32 @@
+:root {
+  --color-primary: #6d6afe;
+  --color-red: #ff5b56;
+  --color-yellow: #f5e14b;
+  --color-black: #111322;
+  --color-white: #ffffff;
+  --color-gray-100: #f0f6ff;
+  --color-gray-200: #e7effb;
+  --color-gray-300: #ccd5e3;
+  --color-gray-400: #9fa6b2;
+  --color-gray-500: #3e3e43;
+  --color-gary-600: #f5f5f5;
+  --color-bg: #f0f6ff;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Pretendard;
```

P2:
fonts.css 에서 설정하신 fontFace를 사용하고 싶으신거면 Pretend로 입력하셔야 할 것 같아요.

```suggestion
  font-family: Pretend;
```